minimesos/src/main/java/com/containersol/minimesos/cluster/MesosCluster.java

```diff
@@ -138,6 +143,9 @@ public void start(int timeoutSeconds) {
 
     /**
      * Prints the state of the Mesos master or agent
+     *
+     * @param out TODO(mwl)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1135)